rest-client/src/main/resources/logback.xml

```diff
@@ -0,0 +1,21 @@
+<configuration debug="true">
```

Comment:
I dunno! Where should this go? Do we want a default configuration with the jar, and then another one to overwrite and log more in test?